packages/combo-box/src/vaadin-combo-box-scroller-mixin.js

```diff
@@ -141,14 +141,20 @@ export const ComboBoxScrollerMixin = (superClass) =>
     }
 
     /**
-     * Requests an update for the virtualizer to re-render items.
+     * Updates the virtualizer's size and items.
      */
     requestContentUpdate() {
-      if (!this.opened) {
+      if (!this.__virtualizer) {
         return;
       }
 
-      this.__virtualizer.update();
+      if (this.items) {
+        this.__virtualizer.size = this.items.length;
```

Comment:
If this is supported use case

```js
this.filteredItems.push('bar');
this.requestContentUpdate();
```

this change could actually be considered a fix.

Reply:
If we agree to treat it as a fix, I'll add tests.

Reply:
Added tests, changed `refactor:` to `fix:`.